src/main/java/store/onlinebookstore/mapper/ShoppingCartMapper.java

```diff
@@ -0,0 +1,39 @@
+package store.onlinebookstore.mapper;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+import store.onlinebookstore.config.MapperConfig;
+import store.onlinebookstore.dto.cartitem.CartItemResponseDto;
+import store.onlinebookstore.dto.shoppingcart.ShoppingCartDto;
+import store.onlinebookstore.model.CartItem;
+import store.onlinebookstore.model.ShoppingCart;
+import store.onlinebookstore.model.User;
+
+@Mapper(config = MapperConfig.class)
```

Comment:
create interface CartItemMapper
```suggestion
@Mapper(config = MapperConfig.class, uses = CartItemMapper.class)
```